src/openrct2/interface/Window.h

```diff
@@ -173,15 +173,15 @@ struct rct_viewport
  */
 struct rct_scroll
 {
-    uint16_t flags;          // 0x00
-    uint16_t h_left;         // 0x02
-    uint16_t h_right;        // 0x04
-    uint16_t h_thumb_left;   // 0x06
-    uint16_t h_thumb_right;  // 0x08
-    uint16_t v_top;          // 0x0A
-    uint16_t v_bottom;       // 0x0C
-    uint16_t v_thumb_top;    // 0x0E
-    uint16_t v_thumb_bottom; // 0x10
+    uint16_t flags{};          // 0x00
```

Comment:
This and all similar changes are there to ensure they are zero initialized?

Reply:
Yeah, although I could revert it now as this was when I wanted to try and do stuff in the sub class constructor before realising that was still too early and needed to make OnOpen. The unions make it difficult to fully zero the struct with initialisers.